web/problems/templates/python/check.py

```diff
@@ -84,23 +90,42 @@ def secret(x, hint=None, clean=None):
         Check.current_part["secret"].append((str(clean(x)), hint))
 
     @staticmethod
-    def equal(expression, expected_result, clean=None, env=None, update_env=None):
+    def equal(
+        expression,
+        expected_result,
+        clean=None,
+        env=None,
+        update_env=None,
+        time_limit=None,
```

Comment:
Po mojem ni treba, da je vsaka funkcija opremljena z `time_limit`. Kdor želi, lahko teste napiše kot
```
with Check.time_limit(10):
    Check.equal(...)
    Check.secret(...)
```

Mogoče je to ena ideja za naprej, da bi tudi razne `update_env` in `env` damo v ločene context managerje?